### DISCUSSION TITLE
如何贡献代码，以及是否有代码贡献规范？

### DISCUSSION THREAD

**Discussion by shar0** [ejoy/ant#58] [General]
感觉现在的Ant引擎还处在很早期的阶段，如果可以力所能及的方式参与进来，相信比一个人闷头造轮子会更有趣。

**Comment by cloudwu** [comment]
我们用 Ant 开发的游戏已经基本完成了，从某种意义说，它已经具备了完整的功能。

但另一方面，Ant 还不完善，因为它尚未开发过第二个不同的游戏。

所以，我认为最好的参与就是用它开发一个游戏。在开发过程中产生的需求，拿出来讨论，看看是不是已经支持了，还是需要额外开发。

没有真正游戏开发驱动而臆想出来的东西很容易变成伪需求。
